ctapipe/instrument/InstrumentDescription.py

```diff
@@ -905,6 +907,222 @@ def load_config(filename):
     print('Astropy tables have been created.')
     return(telescope,camera,optics)
 
+
+def override_in_table(table,item={}):
+    # Load by default
+    try:
+        newvalue=item[newvalue]
+    except:
+        newvalue=None
+    try:
+        variable=item[variable]
+    except:
+        variable=None
+    try:
+        entry=item[entry]
+    except:
+        entry = None
+    
+
+    # Case A) no variable, no item, override the whole table
+
+    if newvalue==None:
+        return(table)
+
+    if (variable==None and item==None):
+        table['Data'] = newvalue
+    elif (variable==None and item!=None):
+        if '=' in item:
+            key,val = entry.split('=')
+            table['Data'][table['Data'][key] == val] = newvalue
+        else:
+            table['Data'][entry] = newvalue
+    elif (variable!=None and entry==None):
+        table[variable] = newvalue
+    else:
+        if '=' in entry:
+            key,val = entry.split('=')
+            table['Data'][variable][table['Data'][key] == val] = newvalue
+        else:
+            table['Data'][variable][entry] = newvalue
+
+    return(table)
+        
+
+### OrderedDict are not defined by default, so we have to register it in yaml.
+# to load it from file -> u.Quantity(value=string.split(",")[0], unit=string.split(",")[1])
+def represent_quantity(dumper, data):
+    value = []
+    for element in data:
+        item_key   = float(L.value)
+        item_value = str(L.unit)
+    
+    return dumper.represent_scalar(u'%s,%s' %(item_key,item_value)
+
+def load_yaml(filename):
+    import yaml
+    """
+    Function writing data from YAML files (fill with faked data wherever needed)
+    into an astropy.table Table
+    
+    Return
+    ------
+    telescope,camera,optics: 3 dictionaries
+        all dictionaries contain astropy.table Tables
+    """
+    print('Yaml + Faked data fill will be produced.')
+    version = 'Oct2016'
+    tel_num = 10
+    tel_id = [random.randrange(1,124,1) for x in range(0,tel_num)]
+    
+    telescope = {}
+    camera    = {}
+    optics    = {}
+    auxiliary = {}
+    
+    with open(filename, 'r') as fin:
+        yamlobject = yaml.load(fin)
```

Comment:
One always should use `yaml.safe_load` if the addtional parsing features of `yaml.load` are not needed.
See https://youtu.be/kjZHjvrAS74?t=6m50s.